src/Mypage/components/LecueList/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useState } from 'react';
+
+import { BOOK_LIST, NOTE_LIST } from '../../constants/DATA';
+import { LecueBookType, LecueNoteType } from '../../types/myPageType';
+import LecueBook from '../LecueBook';
+import LecueNote from '../LecueNote';
+import * as S from './LecueList.style';
+
+function LecueList() {
+  const [clickedBtn, setClickedBtn] = useState(true);
+  const [noteCount, setNoteCount] = useState(0);
+  const [bookCount, setBookCount] = useState(0);
```

Comment:
두 state가 비슷하게 동작하는 것 같은데, 아래처럼 수정해보면 어떨까 합니당!!
요렇게하면 두 개의 state를 하나로 줄일 수 있어용!

```suggestion
  const [counter, setCounter] = useState([0, 0]);
```